volume/cache/machine.json

```diff
@@ -1 +0,0 @@
-{"machine_id": "7abfc17e"}
```

Comment:
These files should never have been committed. They're artifacts of running localstack locally. I've updated the .gitignore to prevent these files coming back in the future.